### ISSUE TITLE
Chef 12.4 results in "Cannot find a provider for ruby_install[VERSION] on PLATFORM"

### ISSUE THREAD

**Issue by indirect** [express42-cookbooks/ruby_lwrp#2]
On Chef 12.3, this cookbook works perfectly. On Chef 12.4, this cookbook does not seem to work anymore. :( Instead of working, the Chef run fails with this exception:

```
  * ruby_install[2.2.2] action install

    ================================================================================
    error executing action `install` on resource 'ruby_install[2.2.2]'
    ================================================================================

    argumenterror
    -------------
    cannot find a provider for ruby_install[2.2.2] on ubuntu version 14.04

    resource declaration:
    ---------------------
    # in /var/chef/cache/cookbooks/lemur/recipes/rubies.rb

     20:   ruby_install version do
     21:     action :install
     22:   end
     23: end

    compiled resource:
    ------------------
    # declared in /var/chef/cache/cookbooks/lemur/recipes/rubies.rb:20:in `block in from_file'

    ruby_install("2.2.2") do
      action :install
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      declared_type :ruby_install
      cookbook_name "lemur"
      recipe_name "rubies"
    end
```

**Comment by sample** [comment]
Yes, this is known bug on 12.4, i'll investigate this.

**Comment by sample** [comment]
Fixed in 0.1.7
